doc/pause.md

```diff
@@ -0,0 +1,112 @@
+# Pause Experiment
+
+This document describe how to pause a running chaos in Chaos Mesh.
+
+## Pause
+
+Pause is a state that a running chaos is recovered temporally but not deleted.
+Undoing pausing a chaos will run the chaos again with same parameter.
+
+## How To
+
+For instance we have a podchaos:
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: PodChaos
+metadata:
+  name: pod-kill-example
+  namespace: chaos-testing
+spec:
+  action: pod-kill
+  mode: one
+  selector:
+    namespaces:
+      - chaos-testing
+  duration: "10s"
+  scheduler:
+    cron: "@every 15s"
+```
+
+While the chaos is running we can get its status like below:
+```shell script
+$ kubectl get podchaos pod-kill-example --namespace chaos-testing --output yaml \
+&& kubectl get pods --namespace chaos-testing
+...
+status:
+  experiment:
+    endTime: "2020-03-29T08:26:46Z"
+    phase: Running
+    podChaos:
+    - action: pod-kill
+      hostIP: 172.17.0.2
+      message: delete pod
+      name: chaos-controller-manager-7f67fbcfdc-lz9jh
+      namespace: chaos-testing
+      podIP: ""
+    startTime: "2020-03-29T08:26:52Z"
+  paused: false
+  phase: ""
+NAME                                        READY   STATUS              RESTARTS   AGE
+chaos-controller-manager-7f67fbcfdc-nm6jl   1/1     Running             0          8s
+chaos-daemon-fbpsz                          1/1     Running             0          5m56s
+chaos-daemon-z7p5f                          1/1     Running             0          5m26s
+chaos-daemon-zhz2k                          0/1     ContainerCreating   0          6s
+```
+
+Pause the running chaos:
+```shell script
+$ kubectl patch podchaos pod-kill-example --namespace chaos-testing --type merge --patch 'status:
+  paused: true'
```

Comment:
Why not defined the paused field on spec? I think define the paused field on status is not friendly for user.

Reply:
That's a point I'm not sure about. Firstly I have the field defined in spec, then I think it's a "status" thing and is not supposed to be in the "specification" of chaos, so here it is.
But on the other hand, I don't think user should patch on status, for it's better to be automatically generated/modified.

Reply:
I think `spec` fields are for controller and user, `status` fields are for controller only.

@cwen0 @YangKeao What's your opinion.

Reply:
According to `Deployment`'s spec, I tend to leave paused into spec.

https://github.com/kubernetes/api/blob/b5bd82427fa87d8b6fdf2c0b4cc2a2115c0c6de9/apps/v1/types.go#L303

Reply:
I prefer to set `Paused` field in spec .

Reply:
@oraluben Great ! I think you have already fixed this up. However, the doc should be fixed same either.

Reply:
@oraluben Please update this document.